frb_codegen/src/library/commands/dart_format.rs

```diff
@@ -30,46 +17,8 @@ pub fn dart_format(
         "format",
         "--line-length",
         line_length.to_string(),
-        *paths
+        ".",
```

Comment:
thus maybe we can make a chunked solution like

```
for paths_chunk in make_it_into_chunks(paths)  {
  command_run!(... paths_chunk ...)
}
```

Reply:
That was the initial idea, but then I realized we are applying `dart fix` to the whole output directory too (since it's limited by the command itself), so we could do the same for `dart format`. There is not much pain about the precision here IMO.

Reply:
Ah I see. just a nit: dart fix is optional and users may disable it, while dart format is almost surely should not be disabled (o/w the code will be very ugly)

Reply:
Yes I know dart_format can be disabled, I just mean maybe a large a portion of people will enable dart format, while not that many will enable dart fix

Reply:
Anyway the code LGTM and I will merge it :)